packages/url_launcher/url_launcher/README.md

```diff
@@ -18,23 +18,23 @@ To use this plugin, add `url_launcher` as a [dependency in your pubspec.yaml fil
 import 'package:flutter/material.dart';
 import 'package:url_launcher/url_launcher.dart';
 
-const String _url = 'https://flutter.dev';
+final Uri _url = Uri(scheme:'https', host:'flutter.dev');
```

Comment:
I went back and forth on that. I guess for this simple example I can show it with `parse`, since we have the more complex example later in the README.

My hesitation is that it'll encourage people to blindly replace `load(aStringThatMayNotActuallyBeAValidUrl)` with `loadUrl(Uri.parse(aStringThatMayNotActuallyBeAValidUrl))`, and still make all the same mistakes that led to the many issue reports that prompted me to switch to a `Uri`-based API in the first place. Like using string interpolation to make a `mailto` URL with `subject` and `body` and not encoding them.

But I guess it'll still be an improvement because instead of failing on iOS and probably working on Android, and then them filing a bug saying the plugin is broken, they'll get an exception in their own code (on all platforms), and be much more likely to understand what they need to do differently.